src/Tribe/Views/V2/View.php

```diff
@@ -2299,4 +2299,28 @@ protected function upcoming_events_count() {
 
 		return (int) tribe_events()->where( 'starts_after', $from_date )->found();
 	}
+
+	/**
+	 * Returns the View current URL query arguments, parsed from the View `get_url()` method.
+	 *
+	 * Since there are a number of parties filtering each View URL arguments, this method will
+	 * parse a View URL query arguments from its filtered URL. This will include all the modifications
+	 * done to a View URL by other plugins and add-ons.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string,mixed> The current View URL args or an empty array if the View URL is empty
+	 *                             or not valid..
+	 */
+	public function get_url_args() {
+		$view_url       = $this->get_url( false );
+		$view_query_str = parse_url( $view_url, PHP_URL_QUERY );
```

Comment:
:warning: parse_url() is discouraged because of inconsistency in the output across PHP versions; use wp_parse_url() instead.

<sub>hash: d4d38172446fa78fd92f6042de1e635c</sub>